fe/src/main/coffee/bustracker.coffee

```diff
@@ -74,7 +73,7 @@ getGetRouteForPhone = ->
             map: map
   
         newLatenessSeconds = if (age = toPosition.age) and oldLatenessSeconds
-          Math.max oldLatenessSeconds, age
+          Math.min oldLatenessSeconds, age
```

Comment:
Last report = least aged/late report

This change means that only the red bus shows as late when the server is initially started, which I think is the intended behavior.